packages/unlock-js/src/__tests__/helpers/configureUnlock.js

```diff
@@ -11,6 +11,7 @@ export default async (
   transactionOptions = {},
   callback
 ) => {
+  const { ethers } = hre
```

Comment:
Any reason why you can't do that once at the top of the file?

Reply:
that relates to how vite/vitest import works